packages/validation/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "barnard59-validation",
+  "version": "0.2.0",
+  "type": "module",
+  "main": "cli.js",
+  "bin": {
+    "barnard59-validate": "./cli.js"
+  },
+  "scripts": {
+    "test": "mocha",
+    "posttest": "npm run generate-rules",
+    "generate-rules": "node lib/rules.js > ./rules.json"
+  },
+  "author": "Zazuko GmbH",
+  "license": "MIT",
+  "devDependencies": {
+    "@jsdevtools/chai-exec": "^2.1.1",
+    "barnard59-base": "^1.2.1",
+    "barnard59-core": "^3.0.1",
+    "barnard59-formats": "^1.4.1",
+    "chai": "^4.3.0",
+    "deep-equal": "^2.0.5",
+    "esmock": "^2.3.8",
+    "sinon": "^15.2.0"
+  },
```

I suggest that we also add information about where to find the package, like this:

```suggestion
  },
  "repository": {
    "type": "git",
    "url": "git://github.com/zazuko/barnard59.git",
    "directory": "packages/validation"
  },
  "bugs": {
    "url": "https://github.com/zazuko/barnard59/issues"
  },
  "homepage": "https://github.com/zazuko/barnard59",
```

This is useful to quickly find the repo from npmjs.com and so.